### DISCUSSION TITLE
Deprecation: future is being installed using the legacy 'setup.py install' method

### DISCUSSION THREAD

**Discussion by AlexandreUSA** [emporia-vue-local/esphome#236] [General]
FYI. Note sure if this is important...

<img width="689" alt="EmporiaVue2" src="https://github.com/emporia-vue-local/esphome/assets/56739736/f1574807-7da6-4c63-8660-551b3e9916d8">

**Comment by dmtalon** [comment]
I just updated one of my vue's tonight and noticed the same messaging and wondering the same.

**Comment by flaviut** [comment]
Nothing to worry about here, this is unrelated to this project and someone will probably fix it by the time it becomes a problem.

And in any case, this will never have any impact on the Vue functioning properly, only the build and upload process.

**Comment by AlexandreUSA** [comment]
Thanks for letting us know.
